### PR TITLE
add test for dataset name with whitespace

### PR DESCRIPTION
**Pull request by asdvalenzuela**

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func TestGetSamplerTypes(t *testing.T) {
 		Sampler = "DeterministicSampler"
 		SampleRate = 2
 
-	[SamplerConfig.dataset1]
+	[SamplerConfig.'dataset 1']
```
Comment (martin308) [inline config/config_test.go:110]: might be worth adding a comment to the `rules.toml` to make sure people know about escaping the dataset name if it contains a space
